teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization:, name: 'Unknown', age: 0, parent_permission: true)
```

Comment:
- [ ] As I suggested above that age is not an optional parameter so if you change your Person class so it will affect its child classes too it would be nice if you update this constructor like the below ⬇️

```ruby
def initialize(age, specialization, name = "Unknown", parent_permission = true)
```